gem2rpm/scl.spec.erb

```diff
@@ -33,29 +33,29 @@ Source0: <%= download_path %>%{gem_name}-%{version}.gem
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 <% for req in spec.required_ruby_version -%>
-Requires: %{?scl_prefix_ruby}ruby <%= req %>
+Requires: %{?scl_prefix_ruby}ruby<%= req != '' ? req : '' %>
```

Comment:
Isn't this the same as `<%= req %>` as the code says: if req is not an empty string, print req, otherwise print an empty string?

Reply:
It's the little things... ya know, like the space that is missing. Now it's present.

Reply:
Heh.

I wish we could do something like this (we can't):
```erb
Requires: %{?scl_prefix_ruby}ruby <%=- req %>
```

Reply:
Yea, I'm open to some other code solution but as I keep running into this when trying to update packages I need a solution present in the code base.